data/mods/gen9ssb/conditions.ts

```diff
@@ -663,6 +672,111 @@ export const Conditions: {[k: string]: ModdedConditionData & {innateName?: strin
 		noCopy: true,
 		// phrases TBD
 	},
+	partman: {
+		noCopy: true,
+		onStart(pokemon) {
+			let message;
+			switch (this.toID(enemyStaff(pokemon))) {
+			case 'partman':
+				message = 'Hii Q - oh, it\'s just me.';
+				break;
+			case 'hydrostatics':
+				message = 'Here to bully Hydro';
+				break;
+			case 'softflex':
+				message = '/me softly flexes';
+				break;
+			case 'computerwizard8800':
+				message = 'CWIZ SLEEP';
+				break;
+			case 'krytocon':
+				this.add(`c:|${getName('PartMan')}|%r 14 // @Krytocon`);
+				this.add(`c:|${getName('Ice Kyubs')}|Roll: 14`);
+				message = null;
+				break;
+			case 'kennedy':
+				message = 'Down the reds!';
+				break;
+			case 'mex':
+				message = 'Probopass moment';
+				break;
+			case 'monkey':
+				message = 'Remember to smile!';
+				break;
+			case 'pissog':
+				message = 'Ma ciaomi queste noci';
+				break;
+			case 'sulo':
+				message = '...Sulo\'s AFK again, aren\'t they?';
+				break;
+			case 'warriorgallade':
+				message = 'Berry nice to meet you!';
+				break;
+			case 'breadloeuf':
+				message = 'BREADBOWL';
+				break;
+			case 'pyro':
+				message = 'Fight me you boiled potato';
+				break;
+			case 'rsb':
+				message = '/me hugs';
+				break;
+			case 'zalm':
+				message = '<(:O)00000>';
+				break;
+			case 'trace':
+				this.add('-message', `PartMan's Neutralizing Gas filled the area! (but not really)`);
+				message = null;
+				break;
+			case 'za':
+				message = '/me shitposts';
+				break;
+			case 'arsenal':
+				message = 'Do I count as a gunner?';
+				break;
+			case 'notater517':
+				message = 'E-excuse me s-senpai >///<';
+				break;
+			case 'siegfried':
+				message = 'Is Sieg baked or boiled?';
+				break;
+			case 'smelysocks':
```

Comment:
they also namechanged

Reply:
Resolved in https://github.com/smogon/ps-ssb/pull/86/commits/159f9f40c88ea29721c0486aea62f852a5a384de